app/controllers/games_controller.rb

```diff
@@ -1,10 +1,26 @@
 class GamesController < ApplicationController
-  def new
-  end
+  def new; end
 
-  def create
-  end
+  def create; end
 
   def show
+    draw_board
+  end
+
+  private
+
+  def draw_board
+    @board = Array.new(8) { Array.new(8) }
+    column = [1, 2, 3, 4, 5, 6, 7, 8]
+    row = [1, 2, 3, 4, 5, 6, 7, 8]
+    row.each_with_index do |_row, ri|
+      column.each_with_index do |_column, ci|
+        @board[ri][ci] = if ci.even? && ri.even? || ri.odd? && ci.odd?
+                           'white_field'
+                         else
+                           'black_field'
+                         end
```

Comment:
Always try to avoid abbreviations. Makes reading the code later easier (or other team members).

```Ruby
column.each_with_index do |_column, column_index|
  if column_index.even? && row.even? || row_index.odd? && column_index.odd?
     board[row_index][column_index] = 'white_field'
   else
     board[row_index][column_index] = 'black_field'
   end
end
```